### PR TITLE
Create the User table and input mock data

### LINKED ISSUE #6: Create user database schema

### PR DESCRIPTION
**Pull request by nikhillakshman**

### CODE REVIEW

File: cric-exchange-app-server/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,1183 @@
+{
```
Comment (rohitnair987) [inline cric-exchange-app-server/package-lock.json:1]: Do we generally publish package-lock.json for node projects?

File: cric-exchange-app-server/ormconfig.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "default",
+  "type": "postgres",
+  "host": "localhost",
+  "port": 5432,
+  "username": "postgres",
```
Comment (rohitnair987) [inline cric-exchange-app-server/ormconfig.json:6]: Is this an interim solution before taking it to the cloud?
I'm worried about saving these here

File: cric-exchange-app-server/src/entity/User.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import {Column, PrimaryGeneratedColumn, Entity} from "typeorm";
+
+@Entity()
+export class User {
+
+    @PrimaryGeneratedColumn()
```
Comment (rohitnair987) [inline cric-exchange-app-server/src/entity/User.ts:6]: Wow! Is this like an auto-incrementing id field?

File: cric-exchange-app-server/README.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# How to setup db
+
+1. Install the latest version of Postgres
```
Comment (rohitnair987) [inline cric-exchange-app-server/README.md:3]: A url to Postgres will be helpful

File: cric-exchange-app-server/package.json [hunk 5]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "cric-exchange-app-server",
+  "version": "0.0.1",
+  "description": "App server that handles request from cric exachange UI.",
```
Comment (rohitnair987) [inline cric-exchange-app-server/package.json:4]: cric exachange -> CricExchange

**Review by rohitnair987 — COMMENTED** [review]
Should we name it as app-server instead of cric-exchange-app-server ... we can later name the other one as ui maybe

File: cric-exchange-app-server/src/index.ts [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import {createConnection} from "typeorm";
```
Comment (rohitnair987) [inline cric-exchange-app-server/src/index.ts:1]: Can we rename this file to User.ts?

File: cric-exchange-app-server/src/index.ts [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+import {User} from "./entity/User";
+
+// connection settings are in the "ormconfig.json" file
+createConnection().then(async connection => {
```
Comment (rohitnair987) [inline cric-exchange-app-server/src/index.ts:5]: How is this user creation called from outside this file?

File: cric-exchange-app-server/src/entity/User.ts (hunk 3 above)
Comment (nikhillakshman) [reply cric-exchange-app-server/src/entity/User.ts:6]: Mind === blown

File: cric-exchange-app-server/src/index.ts (hunk 7 above)
Comment (nikhillakshman) [reply cric-exchange-app-server/src/index.ts:5]: The scaffolding will come in a later MR.

File: cric-exchange-app-server/README.md (hunk 4 above)
Comment (nikhillakshman) [reply cric-exchange-app-server/README.md:3]: Done

File: cric-exchange-app-server/src/index.ts (hunk 6 above)
Comment (nikhillakshman) [reply cric-exchange-app-server/src/index.ts:1]: There is already a file name User.ts . Do you mean rename the import?

Comment (rohitnair987) [reply cric-exchange-app-server/src/index.ts:1]: Isn't this index.ts?
but nvm i see that this a temp thing

File: cric-exchange-app-server/package-lock.json (hunk 1 above)
Comment (nikhillakshman) [reply cric-exchange-app-server/package-lock.json:1]: Ideally you always want to publish the lock file to get consistent builds. https://stackoverflow.com/questions/63187000/what-is-the-npm-equivalent-of-yarn-install-frozen-lockfile

**Comment by nikhillakshman** [comment]
> Should we name it as app-server instead of cric-exchange-app-server ... we can later name the other one as ui maybe

Yes. Will rename it as part of the next check in

**Review by rohitnair987 — APPROVED** [review]

**Pull request merged**